src/libcore/char.rs

```diff
@@ -904,5 +904,5 @@ impl<I: Iterator<Item = u8>> Iterator for DecodeUtf8<I> {
     }
 }
 
-#[unstable(feature = "fused", issue = "35602")]
+#[stable(feature = "fused", since = "1.25.0")]
```

Comment:
`DecodeUtf8` is still unstable so this should be `#[unstable(feature = "decode_utf8", issue = "33906")]`.